job-tutorial/cntk/cntk-distributed-example.json

```diff
@@ -0,0 +1,30 @@
+{
+  "jobName": "cntk-distributed-jobguid",
+  "image": "pai.run.cntk",
+
+  // prepare cmudict corpus in CNTK format https://git.io/vbT5A and upload to hdfs
```

Comment:
comments are not supported in standard json?

Reply:
right